synapse/cores/xact.py

```diff
@@ -65,11 +66,11 @@ def spliced(self, act, **info):
         if pdef is not None and pdef[1].get('local'):
             return
 
-        info['act'] = act
         info['time'] = self.tick
         info['user'] = s_auth.whoami()
+        evtname, evtinfo = s_splice.splice(act, **info)
```

Comment:
this makes and unpacks a tuple right away; do we want to do this inline? especially since we'll be calling this ALOT.

Reply:
Makes sense, changed.